src/wp-includes/comment.php

```diff
@@ -484,6 +484,21 @@ function get_comment_meta( $comment_id, $key = '', $single = false ) {
 	return get_metadata( 'comment', $comment_id, $key, $single );
 }
 
+/**
+ * Queue comment meta for lazy-loading.
+ *
+ * @since 6.3.0
+ *
+ * @param array $comment_ids List of comment IDs.
+ */
+function wp_lazyload_comment_meta( array $comment_ids ) {
```

Comment:
This seems very similar to `wp_queue_comments_for_comment_meta_lazyload()`, why is it required?

Reply:
The function `wp_queue_comments_for_comment_meta_lazyload` is now a wrapper for `wp_lazyload_comment_meta`.  `wp_queue_comments_for_comment_meta_lazyload` expects an array of comment objects. `wp_lazyload_comment_meta` use array of ids. We also have [wp_lazyload_term_meta](https://github.com/WordPress/wordpress-develop/blob/5e4d1bc2d65fa3efa4585c6620c12a55d52d57b3/src/wp-includes/taxonomy.php#L1441) and this follows this naming structure.